src/Testing/src/xunit/WindowsVersions.cs

```diff
@@ -51,6 +51,11 @@ public static class WindowsVersions
     /// </summary>
     public const string Win10_20H2 = "10.0.19042";
 
+    /// <summary>
+    /// 21H2, 22000
+    /// </summary>
+    public const string Win11_21H2 = "10.0.22000";
```

Comment:
@Tratcher do we update this file proactively i.e. before we have test code needing to distinguish new Windows versions❔ @AraHaan, if yes, we could do this part even if we rejected all the rest.